### DISCUSSION TITLE
Can react-markdown be used with react-native?

### DISCUSSION THREAD

**Discussion by braincore** [remarkjs/remark#1002] [Q&A] [answered]
Any success stories? The last remark I see about this is from 2018: https://github.com/remarkjs/react-markdown/issues/141

**Comment by ChristianMurphy** [comment, marked as answer]
It is in theory possible.
Likely by using a similar approach to https://github.com/mohebifar/react-markdown-native, which overrides components with react native alternatives.
This approach could be applied to the latest version of `react-markdown` using the [`components` prop](https://github.com/remarkjs/react-markdown#props), and offering react native alternatives to any HTML elements you want to support.

**Reply by braincore** [reply]
Definitely know it's possible in theory as suggested by the issue I linked to. Wanted to see if there were any success stories out there.

**Reply by ChristianMurphy** [reply]
There are, https://github.com/mohebifar/react-markdown-native is one, though it is no longer maintained.
